modules/oe_content_call_tenders/oe_content_call_tenders.post_update.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for OpenEuropa Call for tenders Content module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Add maxlegnth to oe_content_short_title, oe_summary, oe_teaser, title.
+ */
+function oe_content_call_tenders_post_update_00001() {
+  $file_storage = new FileStorage(drupal_get_path('module', 'oe_content_call_tenders') . '/config/post_updates/00001_add_maxlength');
+  $storage = Drupal::entityTypeManager()->getStorage('entity_form_display');
+
+  $display_id = 'core.entity_form_display.node.oe_call_tenders.default';
+  $values = $file_storage->read($display_id);
+  /** @var \Drupal\Core\Entity\Entity\EntityFormDisplay $form_display */
+  $form_display = $storage->load($values['id']);
+
+  if ($form_display) {
+    $storage->updateFromStorageRecord($form_display, $values);
+    $form_display->save();
+    \Drupal::cache('config')->delete($display_id);
```

Comment:
similar issue found in https://github.com/openeuropa/oe_media/blob/c9fbff9ca4c897510e3bdf3096ec3aa4427d99ae/modules/oe_media_iframe/oe_media_iframe.post_update.php#L93

Reply:
Cache is cleared after processing update.

Reply:
removed